app/javascript/graphql-explorer/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import GraphQLExplorer from './components/GraphQLExplorer';
+import graphQLFetcher from './components/graphQLFetcher';
```

Comment:
Per [ReactJS](https://reactjs.org/docs/jsx-in-depth.html) convention, user defined components must be capitalized.

- `GraphQLExplorer` is a user-defined component and is capitalized


- `graphQLFetcher` looks like an exported function and not a component, and hence is not capitalized.
However, I think it needs to be in a different folder (i.e. not in the `components` folder) - so that we don't confuse it with a `component`

Reply:
Thanks @AparnaKarve ! It was in the back of my mind to ask about this as it didn't seem right, but I wasn't sure what the convention was for exported functions!